packages/logging-controller/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
```

Comment:
I think this needs to include the header and the preamble in order for `yarn changelog:validate` to pass:

```
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

[Unreleased]: https://github.com/MetaMask/core/
```